File: logic build problem/easy problems/python/distance.py
import math
def distance(x1:int,y1:int,x2:int,y2:int)->bool:
    dx = x2 - x1
    dy = y2 - y1
    return math.sqrt((dx * dx) + (dy * dy))
        
print("%.6f"%distance(3, 4, 4, 3))

File: logic build problem/easy problems/Typescript/distance.ts
function distance(x1:number,y1:number,x2:number,y2:number):number{
    const dx = x2 - x1
    const dy = y2 - y1
    return Math.sqrt((dx * dx) + (dy * dy))
}

console.log(distance(3, 4, 4, 3).toFixed(6))
